open_academy/views/menus.xml

```diff
@@ -0,0 +1,6 @@
+<odoo>
+    <menuitem id="menu_root" name="Courses"></menuitem>
+    <menuitem id="menu_list" name="Courses" parent="menu_root"/>
```

Comment:
why do you have so many menu items?  You just need a Menu item for the course and another for the session without a root